x-pack/plugins/security_solution/public/entity_analytics/components/risk_summary_flyout/risk_summary.tsx

```diff
@@ -89,6 +90,30 @@ const RiskSummaryComponent = <T extends RiskScoreEntity>({
     [riskData, telemetry]
   );
 
+  const casesAttachmentMetadata = useMemo(
+    () => ({
+      description: i18n.translate(
+        'xpack.securitySolution.flyout.entityDetails.riskSummary.casesAttachmentLabel',
+        {
+          defaultMessage:
+            'Risk score for {entityType, select, host {host} user {user}} {entityName}',
+          values: {
+            entityName: entityData?.name,
+            entityType: isUserRiskData(riskData) ? 'user' : 'host',
+          },
+        }
+      ),
+      width: 200,
+    }),
+    [entityData?.name, riskData]
+  );
+
+  const timerange = useMemo(() => {
```

Comment:
Since this is a constant we can probably move it outside of the component and skip the hook call altogether

Reply:
The hook call ensures we calculate the time when the component is rendered. Otherwise, it would store the time in which the module was loaded.